src/screens/Categories/index.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import {View, Image, Text, ImageBackground, TouchableOpacity } from 'react-native';
+import styles from './CategoriesStyles';
+import CategoryItem from './../../components/CategoryItem';
+
+export default class Categories extends Component {
+  constructor(props){
+        super(props);
+        this.state = {
+            categories: [
+                { id: 1, title: 'TRABAJO',img: require('./../../images/category-work.png')},
+                { id: 2, title: 'ESCUELA',img: require('./../../images/category-school.png')},
+                { id: 3, title: 'HOGAR',img: require('./../../images/category-home.png')},
+                { id: 4, title: 'AMIGOS',img: require('./../../images/category-friends.png')}
+            ],
+        }
+    }
+
+     renderCat() {
+    const {categories} = this.state;
+    return categories.map(category => {
+      return (
+        <CategoryItem
+          key={category.id}
+          id={category.id}
+          title={category.title}
+          img={category.img}
+          goToTask ={this.goToTask.bind(this)}
+        />
+      )
+    })
+  }
+
+  goToTask(){
+
+      this.props.navigation.navigate('Tasks');
```

Comment:
Una vez que recibas el parametro hay que mandarlo en la funcion de navigate para que pueda ser leido en TaskScreen:

`this.props.navigation.navigate('Tasks', {category} );`

Documentación: https://facebook.github.io/react-native/docs/navigation